client/coral-embed-stream/src/tabs/stream/containers/DraftArea.js

```diff
@@ -71,4 +77,16 @@ DraftAreaContainer.propTypes = {
   disabled: PropTypes.bool,
   rows: PropTypes.number,
   label: PropTypes.string.isRequired,
+  comment: PropTypes.object,
 };
+
+const slots = ['commentBox'];
+
+export default withFragments({
```

Comment:
Also the fragments still needs to related parent fragments leading up to a query.